helm-charts/charts/kube-starrocks/templates/init-pwd/configmap.yaml

```diff
@@ -1,4 +1,12 @@
-{{- if and .Values.initPassword.enabled .Release.IsInstall }}
+{{- if .Values.initPassword.enabled }}
```

Comment:
If remove the `.Release.IsInstall` condition, update fe image, will change Job Spec, this is not allowed by k8s.
`helm upgrade` will fail

Reply:
this is configmap.yaml, not job.yaml. It is fine.